src/components/ui/Footer/Footer.tsx

```diff
@@ -0,0 +1,377 @@
+                alt="Samahan Logo White"
+              />
+            </div>
+            <div
+              id="Text"
+              className="mt-5 xl:mt-10 lg:ml-4 text-center xl:text-left"
+            >
+              <h3 className="text-2xl font-semibold font-inter">
+                Samahan ng mga Mag-aaral ng Pamantasan
+              </h3>
+
+              <h3 className="text-2xl font-semibold font-inter mb-2">
+                ng Ateneo de Davao
+              </h3>
+              <p className="font-inter" style={{ fontSize: '14px' }}>
+                The Student Government of College Unit of the Ateneo de Davao
+                University
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div
+          id="content"
+          className="font-inter flex flex-col lg:flex-row mt-[42px] w-full ml-start mr-auto items-center lg:items-start lg:justify-between"
+        >
+          <div
+            id="left"
+            className="flex flex-col lg:flex-row lg:space-x-[3.125rem]"
+          >
+            {/* Quick Links */}
+            <div
+              id="links"
+              className="flex flex-col space-y-3 text-center lg:text-left mb-[3.125rem] lg:mb-0"
+            >
+              <h3 className="font-bold" style={{ fontSize: '16px' }}>
+                Quick Links
+              </h3>
+              {/* Map Function */}
+              <ul className="space-y-[0.938rem]">
+                {quickLinks.map((link, index) => (
+                  <li key={index}>
+                    <Link href={link.href} legacyBehavior>
+                      <a
+                        className={footerLink({
+                          size: LINK_SIZE_ENUM.REGULAR,
+                          weight: LINK_WEIGHT_ENUM.SUPER_THIN,
+                        })}
+                      >
+                        {link.label}
+                        {link.icon && link.icon}
+                      </a>
+                    </Link>
+                  </li>
+                ))}
+              </ul>
+            </div>
+
+            {/* Offices */}
+            <div
+              id="Offices"
+              className="flex flex-col space-y-3 text-center lg:text-left mb-[3.125rem] lg:mb-0"
+            >
+              <h3 className="font-bold" style={{ fontSize: '16px' }}>
+                Offices
+              </h3>
+              <ul className="space-y-[0.938rem]">
+                {officeLinks.map((link, index) => (
+                  <li key={index}>
+                    <Link href={link.href} legacyBehavior>
+                      <a
+                        className={footerLink({
+                          size: LINK_SIZE_ENUM.REGULAR,
+                          weight: LINK_WEIGHT_ENUM.SUPER_THIN,
+                        })}
+                      >
+                        {link.label}
+                      </a>
+                    </Link>
+                  </li>
+                ))}
+              </ul>
```

Comment:
You can refactor this into something like this to make it cleaner:

`const officeLinks = (
<ul className="space-y-[0.938rem]">
                {officeLinks.map((link, index) => (
                  <li key={index}>
                    <Link href={link.href} legacyBehavior>
                      <a
                        className={footerLink({
                          size: LINK_SIZE_ENUM.REGULAR,
                          weight: LINK_WEIGHT_ENUM.SUPER_THIN,
                        })}
                      >
                        {link.label}
                      </a>
                    </Link>
                  </li>
                ))}
              </ul>
)`

Reply:
Then here under <h3>Offices</h3>, you can call it nalang {officeLinks}

Reply:
You can do this for the rest para cleaner sya. Put it within this file lang before the return statement of this component.